.babelrc

```diff
@@ -4,10 +4,6 @@
     "transform-class-properties",
     "transform-es2015-modules-commonjs",
     "transform-export-extensions",
-    "transform-object-rest-spread",
-    ["transform-async-to-module-method", {
```

Comment:
seems you can remove this package now: `babel-plugin-transform-async-to-module-method`
run `yarn remove babel-plugin-transform-async-to-module-method`

Reply:
removed `babel-plugin-transform-async-to-module-method `and pushed updated `package.json` and `yarn.lock`

Reply:
doesn't seem you pushed yet